### ISSUE TITLE
Reporting current LR and Momentum

### ISSUE THREAD

**Issue by bisraelsen** [pluskid/Mocha.jl#62]
When the docs for "TrainingSummary" say:
```
 Reporting for other solver states like the current learning rate and momentum could be easily added.
```
Does this mean I need to edit the source to add this capability? Or is there a different way?

Thanks!

**Comment by bisraelsen** [comment]
Update:

I modified "src/coffee/training-summary.jl" to print "state.learning_rate" and "state.momentum". I am not sure if this is in your plans but I was thinking I might make a couple parameters for displaying learning rate and momentum and do a pull request.

**Comment by bisraelsen** [comment]
Made more modifications so that TrainingSummary() can accept optional, named, arguments. Submitted pull request to discuss further.
